src/clr-core/icon-shapes/icon.element.ts

```diff
@@ -205,10 +202,3 @@ export class CwcIcon extends IconMixinClass {
   }
 }
 export interface CwcIcon extends IconMixinClass, UniqueId, CssHelpers {}
-registerElementSafely('cwc-icon', CwcIcon);
-
-declare global {
-  interface HTMLElementTagNameMap {
-    'cwc-icon': CwcIcon;
-  }
-}
```

Comment:
Realized this was attempting to register twice. We already register in the `@clr/core/icon` entry point